crates/iroha_executor/src/default.rs

```diff
@@ -1270,6 +1307,28 @@ pub mod trigger {
     ) {
         let trigger = isi.object();
 
+        let trigger_name = trigger.id().name().as_ref();
+        let naming_is_ok = if let Some(tail) = trigger_name.strip_prefix("multisig_accounts_") {
+            let multisig_system: AccountId =
+                // iroha_test_samples::MULTISIG_SYSTEM_ID
+                "ed01201F677E0900C2F633391310D12D155112DF65EDF9DC800D13797CEE5DAF47B890@system"
+                    .parse()
+                    .unwrap();
```

Comment:
if you want to have hardcoded `system` domain and account, then they shouldn't be defined in the `genesis.json`. Rather they should be always implicitly created

Reply:
> Like genesis domain and account, right?

yes

Reply:
Coded them in `iroha_genesis::GenesisBuilder::default`, not the pre-genesis initial state